TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -82,12 +82,25 @@ services:
     image: 'opensearchproject/opensearch:latest'
```

Comment:
Removing all of this file from the diff. These are workarounds for local testing, but not broadly necessary.